bot_code/trainer/utils/threaded_file_downloader.py

```diff
@@ -67,3 +72,9 @@ def create_and_run_workers(self):
         print('ran through all files in ' + str(self.total_time / 60) + ' minutes')
         print('ran through all files in ' + str(self.total_time / 3600) + ' hours')
         print('average time per file: ' + str((self.total_time / max(1, self.total_files))) + ' seconds')
+
+    @staticmethod
+    def is_sequence(obj):

```

Comment:
I have no idea the proper place to put this. It's already in utils?

Reply:
We can leave it here because i do not think it is used outside this class